src/services/subscription-installation-service.ts

```diff
@@ -117,19 +117,13 @@ export const verifyAdminPermsAndFinishInstallation =
 
 const submitSecurityWorkspaceToLink = async (
 	installation: Installation,
-	gitHubInstallationId: number,
-	gitHubServerAppIdPk: number | undefined,
+	subscription: Subscription,
 	logger: Logger
 ) => {
 
 	try {
-		const gitHubAppClient = await createAppClient(logger, installation.jiraHost, gitHubServerAppIdPk, { trigger: "github-configuration-post" });
-
-		logger.info("Fetching info about GitHub installation");
-		const { data: ghInstallation } = await gitHubAppClient.getInstallation(gitHubInstallationId);
-
 		const jiraClient = await JiraClient.getNewClient(installation, logger);
-		await jiraClient.linkedWorkspace(ghInstallation.account.id);
+		await jiraClient.linkedWorkspace(subscription.id);
```

Comment:
Updated linked workspaces to be subscription ids

Reply:
Just to make sure, if user disconnect and reconnect a subscription (but without re-install the app in github org), the security data will be different. 
Is there a place to remove the data upon disconnect?
I think Harmindar have that here https://github.com/atlassian/github-for-jira/pull/2242/files#diff-087bf7a763d243dfd6ae90017a14d41c979f3e27171e77523c9e3561cf0dccb1R85
But that one still use the orgId.

Reply:
I missed that! ill update that to subscription also.